test/parallel/test-child-process-spawnsync-validation-errors.js

```diff
@@ -3,7 +3,7 @@ const common = require('../common');
 const assert = require('assert');
 const spawnSync = require('child_process').spawnSync;
 const signals = require('os').constants.signals;
-const rootUser = common.isWindows ? false : process.getuid() === 0;
+const rootUser = common.isWindows ? false : common.isIBMi || process.getuid() === 0;
```

Comment:
To be consistent, either

```suggestion
const rootUser = !common.isWindows &&
  (common.isIBMi || process.getuid() === 0);
```

or

```suggestion
const rootUser = common.isWindows ? false :
  common.isIBMi ? true : process.getuid() === 0;
```

(Also be sure that the linter passes, lines shouldn’t be longer then 80 characters)